content/_partners/get-started._en.md

```diff
@@ -37,11 +37,12 @@ partnership_steps_body2: >-
 partnership_steps_header3: >-
     Establish an Interagency Agreement (IAA)
 partnership_steps_body3: >-
-    Login.gov is a cost-recoverable federal service, which means we must, by law, charge other agencies for our work. Our partnership and financial engagement will be governed by an Interagency Agreement (IAA). An IAA is a contract between government agencies. For Login.gov, these are contracts we have with other agencies that let them use Login.gov as partners/clients. An IAA may take at least six to eight weeks to complete, which can occur in parallel with the technical integration of your application. <a href="#interagency-agreement-iaa-process" class="caret">Learn more about the IAA process below</a>
+    Login.gov is a cost-recoverable federal service, which means we must, by law, charge other agencies for our work. Our partnership and financial engagement will be governed by an Interagency Agreement (IAA). An IAA is a contract between government agencies. For Login.gov, these are contracts we have with other agencies that let them use Login.gov as partners. <a href="#interagency-agreement-iaa-process" class="caret">Learn more about the IAA process below</a>
 partnership_steps_header4: >-
     Launch integration
 partnership_steps_body4: >-
-     Once testing is complete and the IAA has been executed, Login.gov will launch your integration within two weeks.
+     Once testing is complete and the IAA has been executed, Login.gov aims to launch your integration within two weeks.
```

Comment:
I don't know how I feel about this whole line - it reads as though there's little involvement from the partner and how their (in)actions affect the timeline. Maybe we could talk about whether there's a better way to communicate this (could be another ticket).

Reply:
I hear you on that but it's not something we can easily change since this was an edit that came through from content review :-/

Reply:
But yes - we could make another ticket to clarify perhaps.